src/components/slider/slider.tsx

```diff
@@ -1396,4 +1406,13 @@ export class Slider implements LabelableComponent, FormComponent, InteractiveCom
     const handleBounds = handle.getBoundingClientRect();
     return intersects(maxLabelBounds, handleBounds);
   }
+
+  /**
+   * Returns a string representing the label value that is parsed for comma separator based on whether groupSeparator prop is set to true/false.
+   *
+   * @param input
+   */
+  private determineGroupSeparator = (input: number): string => {
+    return this.groupSeparator ? input.toLocaleString() : input.toLocaleString().replace(",", "");
```

Comment:
question: Does `input.toLocaleString()` always have commas or could there be other number separators for different languages?

Reply:
Can you use [`localizeNumberString` from `src/utils/locale.ts`](https://github.com/Esri/calcite-components/blob/master/src/utils/locale.ts#L106) instead? It should only be used if `groupSeparator` is true and return the value as is otherwise. +@eriklharper, @benelan for additional hints

Reply:
> commas

 Yeah, good point. A period is used as the thousands separator in many non-English speaking countries.